wrapper/minecraft/src/main/kotlin/org/quiltmc/qkl/wrapper/minecraft/math/Vec2fOps.kt

```diff
@@ -25,66 +25,39 @@ import net.minecraft.client.util.math.Vector2f
  */
 public operator fun Vec2f.plus(other: Vec2f): Vec2f {
     return Vec2f(
-        this.x + other.x, 
+        this.x + other.x,
         this.y + other.y
     )
 }
 
-/**
- * Adds a Float to a [Vec2f].
- */
-public operator fun Vec2f.plus(other: Float): Vec2f {
-    return Vec2f(
-        this.x + other, 
-        this.y + other
-    )
-}
-
 /**
  * Subtracts a [Vec2f] from a [Vec2f].
  */
 public operator fun Vec2f.minus(other: Vec2f): Vec2f {
     return Vec2f(
-        this.x - other.x, 
+        this.x - other.x,
         this.y - other.y
     )
 }
 
-/**
- * Subtracts a Float from a [Vec2f].
- */
-public operator fun Vec2f.minus(other: Float): Vec2f {
-    return Vec2f(
-        this.x - other, 
-        this.y - other
-    )
-}
-
 /**
  * Multiplies a [Vec2f] and a [Vec2f].
  */
 public operator fun Vec2f.times(other: Vec2f): Vec2f {
     return Vec2f(
-        this.x * other.x, 
+        this.x * other.x,
         this.y * other.y
     )
 }
 
-/**
```

Comment:
er, __multiplying__ vectors by a scalar _is_ a defined operation—it just scales both axes. yes it's kinda weird, but it's like scaling the entire vector by a scaling factor, and that _does_ make sense.

Reply:
Alright then, lol, ill readd that 😅